test/e2e/storage/utils/utils.go

```diff
@@ -338,7 +338,7 @@ func StartExternalProvisioner(c clientset.Interface, ns string, externalPluginNa
 			Containers: []v1.Container{
 				{
 					Name:  "nfs-provisioner",
-					Image: "quay.io/kubernetes_incubator/nfs-provisioner:v2.2.0-k8s1.12",
```

Comment:
unsure if we need this. I think we want to move to nfs-client provisioner instead of nfs-provisioner due to ganesha bugs: https://github.com/kubernetes/kubernetes/issues/73535